src/connector/src/common.rs

```diff
@@ -517,6 +517,10 @@ pub struct NatsCommon {
     pub server_url: String,
     #[serde(rename = "subject")]
     pub subject: String,
+    #[serde(rename = "filter_subject")]
+    pub filter_subject: Option<String>,
+    #[serde(rename = "durable")]
+    pub durable: Option<String>,
```

Comment:
```suggestion
    #[serde_as(as = "DisplayFromStr")]
    pub durable: Option<bool>,
```

Reply:
BTW, if durable is set to true, does it mean that the messages will remain forever?

Reply:
`durable` just remembers the position of the ack floor when the consumer is no longer active. For our use case we don't need `InactiveThreshold` since its just the floor and consumer config not all the messages. I can add it if you think its needed though 👍🏽

Reply:
Thanks 👍  It'd be fine. We can add it later if we see the needs.